src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -217,7 +217,7 @@ import './Menu.css'
           {{/menu-item-main}}
         {{/menu-item}}
       {{/menu-list-item}}
-      {{#> menu-list-item}}
+      {{#> menu-list-item menu-list-item--IsGroup=true}}
```

Comment:
Axe dev tools is throwing an error because of this on the flyout examples. We shouldn't need the group role on these `li`'s